instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/LongRoutesSanityTest.kt

```diff
@@ -26,7 +26,7 @@ class LongRoutesSanityTest : BaseCoreNoCleanUpTest() {
     }
 
     @Test
-    fun requestAndSetLongRouteWithoutOnboardTiles() = sdkTest(timeout = 60_000) {
+    fun requestAndSetLongRouteWithoutOnboardTiles() = sdkTest(timeout = 120_000) {
```

Comment:
We already wait for quite a long time for the imstrumentation-tests build to finish on every PR, and API 23 is always the last one. With enabling this test we're increasing this time even more.
What if we disable it on API <=24, for example? The goal here is to test that the route is built, we test it on modern devices in 14 seconds. I'm not sure whether we additionally need to test the same logic on API 23 (more precisely, whether it's worth increasing execution time).
If we want to test performance of building long routes, we have perf-tests for that.
WDYT?

Reply:
>  it's worth increasing execution time

In my opinion it's worth. It's just a sanity check that old devices can handle long routes which SDK provides as a feature.

Reply:
21 minute...
We'd have this sanity check in perf tests. We run them on old devices, don't we?
I actually have an idea wrt a different test suite, I'll bring it up during standup.

Reply:
Sure, I was just suggesting another thing: do not run this test on old devices until we have a different test suite. :)